### DISCUSSION TITLE
blog/graphql-yoga-v3

### DISCUSSION THREAD

**Discussion by giscus** [the-guild-org/the-guild-website#935] [Docs Discussions]
# blog/graphql-yoga-v3

The new version of GraphQL Yoga is out! Learn what has improved and what is new!

https://the-guild.dev/blog/graphql-yoga-v3

**Comment by joeynimu** [comment]
Thank you for all the efforts in making working with GraphQL even easier. It's nice we have docs on Federation, I think it'd be nice to add some docs on our friend schemsStitching. Happy to help in setting up some examples and maybe a blog post too!

**Comment by o-az** [comment]
Amazon work 🎉
